.gitlab-ci.yml

```diff
@@ -131,7 +146,7 @@ Container:
   stage: rpmbuild
   extends: .terraform
   rules:
-    - !reference [.build_rules, rules]
+    - !reference [.ga/eus_rules_all, rules]
```

Comment:
Remove this change and keep the old rules.

We want to build the container image on pull requests to make sure that the build isn't broken, but we don't need the container when exercising the pipelines against a nightly RHEL compose or against the GA composes.